config/crds/cluster_v1alpha1_machine.yaml

```diff
@@ -15,6 +15,10 @@ spec:
     description: Machine status such as Terminating/Pending/Running/Failed etc
     name: Phase
     type: string
+  - JSONPath: .status.addresses
+    description: Access IP(s) of the machine
+    name: Addresses
+    type: string
```

Comment:
Will this print out correctly as a string, given the type is `[]corev1.NodeAddress`?